web/docs/tql2/operators/tail.md

```diff
@@ -0,0 +1,33 @@
+# tail
+
+Limits the input to the last `N` events.
+
+```
+tail [N:uint]
```

Comment:
I think the upper case here sticks out like a sore thumb. Let's just use lower case everywhere.